localization/pose_estimator/ndt_scan_matcher/ndt_scan_matcher/README.md

```diff
@@ -0,0 +1,61 @@
+# ndt_scan_matcher
+
+## Purpose
+
+ndt_scan_matcher is a package for position estimation using the NDT scan matching method.
+
+There are two main functions in this package:
+
+- estimate position by scan matching
+- estimate initial position via ros service using the Monte Carlo method
```

Comment:
I think it is better to write `the ROS service`

Reply:
OK! Thank you! https://github.com/tier4/AutowareArchitectureProposal.iv/pull/690/commits/dc74879f1c2326301855e4f44649bbba1332a692